local-tests/.pylintrc

```diff
@@ -0,0 +1,13 @@
+[MESSAGES CONTROL]
+
+disable=missing-docstring,
+        too-many-instance-attributes,
+        duplicate-code,
+        import-error
```

Comment:
I would remove `import-error` from here and set the pythonpath if that is the problem this disable try to help with

Reply:
done